source/extensions/quic_listeners/quiche/platform/quic_file_utils_impl.cc

```diff
@@ -0,0 +1,55 @@
+// NOLINT(namespace-envoy)
+
+// This file is part of the QUICHE platform implementation, and is not to be
+// consumed or referenced directly by other Envoy code. It serves purely as a
+// porting layer for QUICHE.
+
+#include "extensions/quic_listeners/quiche/platform/quic_file_utils_impl.h"
+
+#include "common/filesystem/directory.h"
+#include "common/filesystem/filesystem_impl.h"
+
+#include "absl/strings/str_cat.h"
+
+namespace quic {
+namespace {
+
+void DFTraverseDirectory(const std::string& dirname, std::vector<std::string>& files) {
```

Comment:
Naming?  I'm not entirely sure what DF stands for here.

Reply:
Rename to depthFirstTraverseDirectory